src/BizHawk.Client.Common/config/PathEntryCollection.cs

```diff
@@ -3,16 +3,17 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Reflection;
+using System.Text.Json;
+using System.Text.Json.Serialization;
 
 using BizHawk.Common.CollectionExtensions;
 using BizHawk.Common.PathExtensions;
 using BizHawk.Emulation.Common;
 
-using Newtonsoft.Json;
-
 namespace BizHawk.Client.Common
 {
-	[JsonObject]
+	[JsonConverter(typeof(PathEntryCollectionConverter))]
 	public class PathEntryCollection : IEnumerable<PathEntry>
```

Comment:
Again we can drop the interface, though this one was actually used. I've done both in https://github.com/TASEmulators/BizHawk/commit/75502e7ffb5be5af8c23d52ad446b10b0e8efc1d.